containers/package-builder/build-concent-api-docs.sh.j2

```diff
@@ -10,7 +10,7 @@ cp --recursive "$concent_dir/docs" "$build_dir/concent-api-docs"
 
 cd "$build_dir/concent-api-docs/docs/"
 
-virtualenv --python python3 "$build_dir/docs-virtualenv/"
+virtualenv --python python{{ python_version }} "$build_dir/docs-virtualenv/"
```

Comment:
Please put `python{{ python_version }}` in quotes.

Do this here an in other places in this pull request.

Reply:
Done.